src/main/drivers/accgyro_mpu6500.c

```diff
@@ -97,8 +97,8 @@ void mpu6500GyroInit(uint16_t lpf)
     mpuConfiguration.write(MPU_RA_PWR_MGMT_1, INV_CLK_PLL);
     mpuConfiguration.write(MPU_RA_GYRO_CONFIG, INV_FSR_2000DPS << 3);
     mpuConfiguration.write(MPU_RA_ACCEL_CONFIG, INV_FSR_8G << 3);
-    mpuConfiguration.write(MPU_RA_CONFIG, mpuLowPassFilter);
-    mpuConfiguration.write(MPU_RA_SMPLRT_DIV, 0); // 1kHz S/R
+    mpuConfiguration.write(MPU_RA_CONFIG, lpf);
+    mpuConfiguration.write(MPU_RA_SMPLRT_DIV, gyroMPU6xxxGetDividerDrops()); // Get Divider
```

Comment:
suggest renaming `gyroMPU6xxxGetDividerDrops` to `gyroMPU6xxxCalculateDivider` and removing comment.